drivers/bridge/bridge.go

```diff
@@ -376,7 +376,8 @@ func (d *driver) configure(option map[string]interface{}) error {
 				logrus.Warnf("Running modprobe bridge br_netfilter failed with message: %s, error: %v", out, err)
 			}
 		}
-		removeIPChains()
+		// TODO: need a better way to handle this
+		// removeIPChains()
```

Comment:
I think this change is for avoiding network disruption during the reload, but I am worried about getting rid of this call, which let us start from a clean state after ungraceful daemon shutdowns.

I heard the daemon restart use-case is narrowed to only the time needed to update the daemon and not meant to allow containers running for long time w/o a daemon. If that is the case, maybe we can live with an iptables cleanup during daemon restart. The rules are anyway reconstructed at the time the driver loads the networks.